.github/workflows/osv-scanner.yml

```
@@ -42,3 +48,9 @@ jobs:
         -r
         --skip-git
         ./
+    steps:
```

actionlint (1.7.4)

when a reusable workflow is called with "uses", "steps" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "scan-pr"

(syntax-check)